lib/common.js

```diff
@@ -519,109 +527,140 @@ async function notify_audits(payload, username) {
           space: body.space.name,
           received_at: body.received_at,
           username: body.username,
-          info: JSON.stringify(body)
-        }
+          info: JSON.stringify(body),
+        },
       });
     }
   } catch (err) {
-    console.error("An error occured auditing request:", err);
+    console.error('An error occured auditing request:', err);
   }
 }
 
 function socs(envs) {
-  let blacklist_wri_regex = /([A-z0-9\-\_\.]+\:)[A-z0-9\-\_\.]+(\@tcp\([A-z0-9\-\_\.]+\:[0-9]+\)[A-z0-9\-\_\.\/]+)/;
-  let blacklist_uri_regex = /([A-z]+\:\/\/[A-z0-9\-\_\.]*\:)[A-z0-9\-\_\.\*\!\&\%\^\*\(\)\=\+\`\~\,\.\<\>\?\/\\\:\;\"\'\ \t\}\{\[\]\\|\#\$}]+(\@[A-z0-9\-\_\.\:\/]+)/;
-  if(!config.envs_blacklist || config.envs_blacklist === '') {
+  const blacklist_wri_regex = /([A-z0-9\-_.]+:)[A-z0-9\-_.]+(@tcp\([A-z0-9\-_.]+:[0-9]+\)[A-z0-9\-_./]+)/;
+  const blacklist_uri_regex = /([A-z]+:\/\/[A-z0-9\-_.]*:)[A-z0-9\-_.*!&%^*()=+`~,.<>?/\\:;"' \t}{[\]\\|#$}]+(@[A-z0-9\-_.:/]+)/;
+  if (!config.envs_blacklist || config.envs_blacklist === '') {
     return envs;
   }
   try {
-    let blacklist = config.envs_blacklist.split(',');
-    Object.keys(envs).forEach(function(env) {
-      blacklist.forEach(function(blEnv) {
-        if(blEnv && blEnv !== '' && env && env !== '' && env.toLowerCase().trim().indexOf(blEnv.toLowerCase().trim()) > -1) {
+    const blacklist = config.envs_blacklist.split(',');
+    Object.keys(envs).forEach((env) => {
+      blacklist.forEach((blEnv) => {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Arrow function has a complexity of 11. Maximum allowed is 8.](https://app.codacy.com/manual/Akkeris/controller-api/pullRequest?prid=5244943)